en/docs/authentication-and-authorization/secure-communication-between-the-choreo-gateway-and-your-backend.md

```diff
@@ -0,0 +1,130 @@
+# Secure Communication Between the Choreo Gateway and Your Backend
+
+To establish secure communication between the Choreo Gateway and your backend, you can configure mutual SSL.
+
+Mutual SSL authentication involves both the client and server validating each other’s certificates before establishing a connection. The following diagram depicts this scenario:
+
+![Mutual SSL authentication](../assets/img/authentication-and-authorization/mutual-ssl-authentication.png)
+
+## Configure mutual SSL to establish secure connectivity
+
+To establish secure connectivity between the Choreo Gateway and your backend using mutual SSL, you must add the certificate of the backend (server certificate) to Choreo and add the certificate of Choreo (client certificate) as a trusted certificate in the backend.
+
+### Step 1: Configure TLS for a self-signed certificate
+
+!!! info "Note"
+     This step is applicable only under the following conditions:
+
+       - Your backend uses a self-signed certificate.
+       - Your backend employs a certificate signed by an organization-wide certificate authority, which is not recognized as a well-known certificate authority, for enabling TLS.
+
+     If your backend uses valid CA-signed certificates, you can skip this step. 
```

Comment:
This is kind of an ideal situation but due to this issue https://github.com/wso2-enterprise/choreo/issues/28385 we cannot skip backend certificate configuration when configuring MTLS. I think we'll have to remove this note for now until we fix this issue.